src/mscorlib/src/System/Environment.cs

```diff
@@ -369,6 +369,9 @@ public static String CurrentDirectory
             [System.Security.SecuritySafeCritical]  // auto-generated
 #endif
             get {
+#if PLATFORM_UNIX
+                return Path.DirectorySeparatorCharAsString; // TODO: anything better to return?
```

Comment:
It's not used in our corpus so compat with Mono may not be too important.

---
In reply to: [68575619](https://github.com/dotnet/coreclr/pull/6006#discussion_r68575619) [](ancestors = 68575619)